assets/src/components/propertiesPanel.tsx

```diff
@@ -21,16 +23,26 @@ export const hideMeIfNoCrowdingTooltip = (hideMe: () => void) => {
 
 const PropertiesPanel = ({ selectedVehicleOrGhost }: Props) => {
   const [, dispatch] = useContext(StateDispatchContext)
+  const { socket } = useSocket()
+  const liveVehicle = useVehicleForId(socket, selectedVehicleOrGhost.id)
```

Comment:
If we're pulling the `liveVehicle` here in the VPP itself, what's the rationale behind having the full vehicle in the state, rather than just the ID? Is it just a performance thing, so there isn't an initial lag while it fetches the `liveVehicle` for the first time?

(Maybe something further down the PR will illuminate this decision, but I wanted to flag it now while it's fresh in my mind).

Reply:
> Is it just a performance thing, so there isn't an initial lag

That's the basic idea. You have a perfectly good vehicle here in memory, so you can use that as your initial state before the whole subscribe-to-vehicle/respond-with-vehicle-data cycle completes.